react/features/display-name/functions.js

```diff
@@ -11,3 +13,13 @@ export function appendSuffix(displayName: string, suffix: string = '') {
     return `${displayName || suffix}${
         displayName && suffix && displayName !== suffix ? ` (${suffix})` : ''}`;
 }
+
+/**
+ * Selector for whether we are currently in tile view.
+ *
+ * @param {Object} state - The redux state.
+ * @returns {boolean}
+ */
+export function isLayoutTileView(state: Object) {
```

Comment:
Wouldn't it be better if you define the selector in the `video-layout` feature? WDYT?

Reply:
Good point! Moved it.